app/policies/job_policy.rb

```diff
@@ -10,4 +10,8 @@ def show?
   def approve?
     user.has_role?(:admin) || Chapter.find_roles(:organiser, user).any?
   end
+
+  def unpublish?
+    user.has_role?(:admin) || Chapter.find_roles(:organiser, user).any?
```

Comment:
Organisers should not be able to manage jobs so even though we have not fixed the rest of the policies to reflect that I think we should make sure to remove it from any new functionality.

Reply:
@despo ok, sure, can update the new policy.